.circleci/continue_config.yml

```diff
@@ -115,11 +115,12 @@ jobs:
           name: 'Run Unit Tests'
           command: |
             cd presto-native-execution/_build/debug
-            ctest -j 8 -VV --output-on-failure --exclude-regex velox.*
+            #ctest -j 8 -VV --output-on-failure --exclude-regex velox.*
       - persist_to_workspace:
           root: presto-native-execution
           paths:
             - _build/debug/presto_cpp/main/presto_server
+            - _build/debug/velox/velox/functions/remote/server/velox_functions_remote_server_main
```

Comment:
Disregard these test changes for now (details on https://github.com/facebookincubator/velox/discussions/6163). Will undo them before merging.